lib/createMd.js

```diff
@@ -40,7 +40,7 @@ function showState(state) {
 
     line.push(state.indicator ? 'X' : ' ');
     //line.push(state.noSubscribe ? 'NS' : '');
-    line.push(state.role || '');
+    line.push(state.role ? '`' + state.role.replace(/\|/g, '\|') + '`' : ''); //escape | character inside table cell!
```

Comment:
Good idea to print it as code (with backticks), but then we should (just to be safe) also escape the backticks in the replace.